docs/UserGuide.md

```diff
@@ -359,40 +395,140 @@ A person can have any number of tags and insurance policies (including 0).
+      <br><br>
+* To batch delete more than 1 client contact.
+    * `batch delete 1, 2, 4`
+      <br><br>
+      [Return to Table of Contents](#table-of-contents)
+<br><br>
+
+### <span style="color:#b573c9"><code>meet</code>: Schedule a meeting with a client</span>
+
+**Purpose**: Schedules a meeting on a particular date, start time, end time and place with a client in ClientBook.
+
+**Format**: `meet INDEX [-ACTION] DATE START END PLACE`
+
+* Schedules a meeting with the client at the specified `INDEX`.
+* `INDEX` refers to the index number shown in the displayed client list.
+* `INDEX` must be 1 or higher, and less than or equal to the index of the last item in the displayed list.
+* `ACTION` can be `add` to add a meeting, `delete` to delete a meeting, `clear` to clear all meetings.
+* If `-ACTION` is empty, the default action is to add a meeting.
+* `DATE` must be in the `DD:MM:YYYY` format.
+* `START` and `END` must be in the `HH:MM` format.
+* `END` must be after `START` on the same `DATE`.
+* `PLACE` cannot be empty.
+* There is a check for clashes between meetings when adding a new meeting.
+
+**Examples**:
+* Add a meeting and there are no clashes.
+    * `meet 1 20.05.2021 15:00 16:00 MRT`
+
+      ![meet-add](images/meet-add.png)
+
+* Add a meeting but there are clashes.
+    * `meet 3 -add 20.05.2021 15:30 17:30 MRT`
+
+      ![meet-clash](images/meet-clash.png)
+
+* Delete a meeting.
+    * `meet 1 -delete 20.05.2021 15:00 16:00 MRT`
+
+* Clear all meetings.
+    * `meet 2 -clear`
+
+[Return to Table of Contents](#table-of-contents)
+<br><br>
 
-**Format**: `list [-n] [-p] [-e] [-a] [-i] [-t] [-m]`
+### <span style="color:#f75c66">Contact Viewing Features</span>
+Contact viewing features allow you to view your client contacts in different ways. For example,
+you may reorder your contact list or look for specific clients.
+
+### <span style="color:#f75c66"><code>list</code>: List all clients</span>
+
+**Purpose**: Shows a list of all clients in ClientBook. You may use optional identifiers in conjunction with the minus (-)
+symbol to select which client details you want to see, so that you can have a clutter free view of your client contacts.
+
+**Format**: `list [-p] [-e] [-a] [-i] [-m]`
 
 **Examples**: 
 *  `list` without any specified identifiers shows a list of all clients and all their information.
-  * `list`
-    
-    ![list](images/list.png)
+
+![list](images/list.png)
     
-*  One or more identifiers can be used to make `list` only show the specified information. The following command shows a list of all clients and their phone number and insurance policy number.
-   * `list -p -i`
+*  One or more identifiers can be used to make `list` only show the specified information. The following command 
+   shows a list of all clients and their phone number and insurance policy number. A client's name and tags will
+   be shown regardless of the identifiers that you specify.
+   
+`list -p -i`
     
```

Comment:
I think this should be indented to the 2nd level, if you compare with examples from the other commands.

Reply:
Done.